File: 백준/Bronze/4153. 직각삼각형/직각삼각형.cc
#include <iostream>
using namespace std;

int a, b, c;

int main()
{
    ios::sync_with_stdio(0);
    cin.tie(0);
    cout.tie(NULL);
    
    while (1) {
        cin >> a >> b >> c;

        if (a + b + c == 0) break;
        
        a = a * a;
        b = b * b;
        c = c * c;

        if (a + b == c || a + c == b || b + c == a) {
            cout << "right" << "\n";
        }
        else {
            cout << "wrong" << "\n";
        }
    }

    return 0;
}

File: 백준/Bronze/4153. 직각삼각형/직각삼각형.py
while True:
    l = list(map(int, input().split()))
    
    if l.count(0) == 3:
        break
    
    l.sort()

    if (l[0] ** 2) + (l[1] ** 2) == l[2] ** 2:
        print('right')
    else:
        print('wrong')